networkx/algorithms/chordal.py

```diff
@@ -344,3 +347,79 @@ def _connected_chordal_graph_cliques(G):
                 raise nx.NetworkXError("Input graph is not chordal.")
         cliques.add(frozenset(clique_wanna_be))
         return cliques
+
+
+@not_implemented_for('directed')
+def complete_to_chordal_graph(G):
+    r"""Complete to chordal graph
+
+        Enhances the graph by edges to receive a chordal graph.
+
+        Parameters
+        ----------
+        G : NetworkX graph
+            Undirected graph
+
+        Returns
+        -------
+        H : NetworkX graph
+            The chordal enhancement of G
+        alpha : Dictionary
+                The elimination ordering of nodes of G
+
+        Notes
+        ------
+        A graph G=(V,E) is called chordal if for each cycle with
+        length bigger than 3, two non adjacent nodes are connected
+        by an edge (called chord). There are different approaches
+        to calculate the chordal enhancement of a graph.
+        The algorithm used here is called MCS-M and gives a minimal
```

Comment:
The difference between them is, that you can have a minimum triangulation, where you cannot remove any edge to contain the property, but the triangulation must not be the minimal. For a minimal triangulation hold, that no other triangulation is smaller in terms of subset.

Reply:
But the docs say it must not be minimum.  Here you say it must not be minimal.
Is there a way to say it without using either "minimal" or "minimum" -- or defining minimal and minimum as you use them?

Reply:
I have done it wrong in this thread. In the docs it is correct. I have there inserted after minimal the term local and after minimum, global. I think this will solve the confusion. You can see it as local and global minimum.

Reply:
Good -- Thanks that is more clear. 

One more language issue. The wording says "this triangulation must not be minimum (global)". I think it should be: "this triangulation is not necessarily minimum (global)".
The first says there is no way it will be minimum (global). The second says it might or might not be minimum (global).

Reply:
Thank you for the correction. I am not a native speaker. The first sentence says now the algorithm returns at least a minimal solution.